src/unix_term.rs

```diff
@@ -18,7 +17,7 @@ pub use crate::common_term::*;
 pub const DEFAULT_WIDTH: u16 = 80;
 
 #[inline]
-pub fn is_a_terminal(out: &Term) -> bool {
+pub fn is_a_terminal<T: AsRawFd>(out: &T) -> bool {
```

Comment:
Nit: prefer to spell this as `out: &impl AsRawFd` to avoid the indirection via `T`.

Reply:
Done, though note `pub(crate) fn set_title<T: Display>(title: T)` at the bottom of the file.